examples/with-typescript/tsconfig.json

```diff
@@ -13,5 +13,6 @@
     "removeComments": false,
     "preserveConstEnums": true,
     "sourceMap": true
+    "noEmit": true
```

Comment:
Sorry I can't accept this.
This option means "No generation .js files" so nextron can't start process.

Reply:
If you want to avoid that error, you should specify "include" or "exclude" options in the `tsconfig.json`.
Because nextron uses `webpack` and `ts-loader`, it's unnecessary to use these options.

Reply:
I updated it to exclude a few of the JS files, seems to work ok.